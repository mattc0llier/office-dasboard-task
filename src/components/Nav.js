import React from 'react';

const Nav = () => {
    return(
        <h2>Office Dashboard</h2>
    )
}

export default Nav