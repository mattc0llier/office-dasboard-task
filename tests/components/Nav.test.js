import React from 'react'
import Nav from '../../src/components/Nav'
import renderer from 'react-test-renderer'

describe('Nav', () => {
    test('matches the Nav snapshot', () => {
        const tree = renderer.create(<Nav />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})