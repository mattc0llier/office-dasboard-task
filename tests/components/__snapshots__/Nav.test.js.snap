// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav matches the Nav snapshot 1`] = `
<h2>
  Office Dashboard
</h2>
`;
